stable/aws-load-balancer-controller/values.yaml

```diff
@@ -32,13 +32,15 @@ rbac:
 podSecurityContext:
   fsGroup: 65534
 
-securityContext: {}
+securityContext:
   # capabilities:
   #   drop:
   #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1002
```

Comment:
let's leave runAsUser and runAsGroup unset. (so the docker file controls it).
we only need runAsNonRoot: true to enforce we are not running as root.

Reply:
sounds good